core-services/prow/02_config/_plugins.yaml

```diff
@@ -151,26 +151,43 @@ bugzilla:
       target_release: 4.1.z
       validate_by_default: true
     openshift-4.2:
-      dependent_bug_target_release: 4.3.0
+      dependent_bug_target_release: 4.3.z
       target_release: 4.2.z
       validate_by_default: true
     openshift-4.3:
       dependent_bug_target_release: 4.4.0
       target_release: 4.3.z
       validate_by_default: true
+    openshift-4.4:
+      dependent_bug_states:
+      - status: MODIFIED
+      - status: ON_QA
+      - status: VERIFIED
+      - status: RELEASE_PENDING
+      - resolution: ERRATA
+        status: CLOSED
+      target_release: 4.4.0
+      validate_by_default: true
     release-4.1:
       dependent_bug_target_release: 4.2.z
       target_release: 4.1.z
       validate_by_default: true
     release-4.2:
-      dependent_bug_target_release: 4.3.0
+      dependent_bug_target_release: 4.3.z
       target_release: 4.2.z
       validate_by_default: true
     release-4.3:
       dependent_bug_target_release: 4.4.0
       target_release: 4.3.z
       validate_by_default: true
     release-4.4:
+      dependent_bug_states:
```

Comment:
do we care about `dependent_bug_states` without `dependent_bug_target_release`?  I think this is something we'd need only together after we stop fast-forwarding the 4.4 branches and have a divergent 4.5.